modules/react/combobox/lib/ComboboxMenuList.tsx

```diff
@@ -31,13 +52,8 @@ export const ComboboxMenuList = createSubcomponent('ul')({
     <ListBox
       as={Element}
       model={model}
-      background={commonColors.background}
-      borderRadius="zero"
-      padding="zero"
-      marginY="xxs"
-      gap="zero"
-      overflowY="auto"
-      {...elemProps}
+      marginY={cssVar(system.space.x2)}
```

Comment:
Maybe it's better to remove `marginY` and use `marginTop` and `marginBottom`, so it can reduce style prop usage in our code